apps/doji-frontend/src/components/Review/RatingEntry.tsx

```diff
@@ -27,10 +27,7 @@ function ReviewEntry(props: { data: IReview }) {
         <Stack direction="row" spacing="0.5em" alignItems="center">
           {/*TODO Avatar*/}
           <StatusBadge username={props.data.user.username}>
-            <Avatar
-              src="https://mui.com/static/images/avatar/2.jpg"
-              sx={{ width: 24, height: 24 }}
-            />
+            <Avatar src={props.data.user.profilePictureURL} sx={{ width: 24, height: 24 }} />
```

Comment:
`profilePictureURL` can be `null`

handle กรณีที่ผู้ใช้ไม่มี profile ด้วย เป็น 

```tsx
<Avartar src={...} >
 {user.displayname.chatAt(0).toUpperCase()}
</Avartar>
```